PluralKit.Bot/Commands/Groups.cs

```diff
@@ -263,7 +263,7 @@ async Task SetIcon(ParsedImage img)
 
             await AvatarUtils.VerifyAvatarOrThrow(_client, img.Url);
 
-            await ctx.Repository.UpdateGroup(target.Id, new GroupPatch { Icon = img.Url });
+            await ctx.Repository.UpdateGroup(target.Id, new GroupPatch { Icon = img.CleanUrl });
```

Comment:
this will break setting avatars using a URL as text in a bot command - `img.CleanUrl || img.Url`